core/script_language.h

```diff
@@ -119,7 +119,7 @@ class Script : public Resource {
 	virtual void get_script_method_list(List<MethodInfo> *p_list) const = 0;
 	virtual void get_script_property_list(List<PropertyInfo> *p_list) const = 0;
 
-	virtual int get_member_line(const StringName &p_member) const { return 0; }
+	virtual int get_member_line(const StringName &p_member) const { return -1; }
```

Comment:
Why `-1`? Traditionally text files start at line 1, so line 0 already would indicate that it's not valid.